### DISCUSSION TITLE
Ideas for other notification agents?

### DISCUSSION THREAD

**Discussion by sct** [sct/overseerr#255] [Feature Requests]
Overseerr currently only supports **Email** and **Discord** notification agents. The system is pretty flexible and can add almost any other type of notification agent users would like.

What are some of your must have notification agents? Slack? Telegram? Others?

Reply with what you would like to see!

**Comment by kaiguy** [comment]
Would love to see Pushover!

**Reply by aabramovich** [reply]
Pushover is a must.  😄

**Comment by rix1337** [comment]
Highly recommend Telegram

**Comment by gentoosu** [comment]
I would like Slack, please.

**Reply by figlizzard12** [reply]
Slack would be great

**Reply by daretogo** [reply]
Just a comment to mention that if you add Slack support you can claim MatterMost support too -   they're webhook system will support Slack without any change so it'd be a 2-for-1 deal.    Not that Mattermost is all that popular...

**Comment by Ayfteyd7Od** [comment]
Gotify!

**Reply by sonofwatt** [reply]
Would love to see this. I use Gotify for all my containers.

**Comment by hiccuplick** [comment]
If you implement something like [Apprise](https://github.com/caronc/apprise), it would give you access to tons of notification services without having to manage them individually. Just a thought.

**Reply by sct** [reply]
Looks to be written in Python. Not sure if I could implement this? I don't have any experience with using Apprise so maybe I am misunderstanding something.

**Reply by hiccuplick** [reply]
Sorry, I guess I should have linked to the [Apprise API](https://github.com/caronc/apprise-api). You can call Apprise from within your application using something as simple as an HTTP POST if you want. The downside of doing it this way is you'd have to either package Apprise with your application, or have your users run it separately. It might not be the right thing for you. I just like it because it supports nearly any notification type you can think of, and I have no problem running my own Apprise API instance.

**Reply by grehund** [reply]
I was hoping to use an Apprise docker container with Webhooks, but I can't get it to work. This was already a workaround for Overseerr not having an option for Prowl notifications. I guess that's my way of asking for a Prowl option in Overseerr, please?

**Comment by ghost** [comment]
Email, Discord, Slack, Google Chat, Microsoft Teams, Telegram, and Webhook would be awesome!

**Reply by sct** [reply]
We already have email but would love to add the others!

**Reply by samwiseg0** [reply]
I think webhooks are a great idea because having the ability to make custom webhooks will cover a large amount of 3rd party integrations.

**Comment by NicksMacLife** [comment]
I think these are all great but it would be awesome if the admin can modify the email address used to notify a user.

**Reply by TheCatLady** [reply]
You already posted #1891, no need to spam multiple channels with the same request.
